src/Microsoft.AspNetCore.Sockets.Http/HttpConnectionDispatcher.cs

```diff
@@ -389,15 +389,15 @@ private static string GetNegotiatePayload(string connectionId, HttpSocketOptions
                 jsonWriter.WriteStartArray();
                 if ((options.Transports & TransportType.WebSockets) != 0)
                 {
-                    jsonWriter.WriteValue(nameof(TransportType.WebSockets));
+                    WriteTransport(jsonWriter, nameof(TransportType.WebSockets), TransferFormat.Text | TransferFormat.Binary);
```

Comment:
Is this change documented in the protocol spec page?

Reply:
Adding it now.